lib/ts/recipe/accountlinking/types.ts

```diff
@@ -184,10 +177,28 @@ export type RecipeInterface = {
     unlinkAccounts: (input: {
         recipeUserId: string;
         userContext: any;
-    }) => Promise<{
-        status: "OK";
-        wasRecipeUserDeleted: boolean;
-    }>;
+    }) => Promise<
```

Comment:
Should createPrimaryUser also return a status like `"RECIPE_USER_ID_ALREADY_PRIMARY_USER"`? We should do that if there is any case in which we want to differentiate between that status vs the `"OK"` status